bloom_test.go

```diff
@@ -454,6 +454,19 @@ func BenchmarkCombinedTestAndAdd(b *testing.B) {
 		f.TestAndAdd(key)
 	}
 }
+func BenchmarkParallelTest(b *testing.B) {
+        key := make([]byte, 100)
+        for j := 0; j < b.N; j++ {
+                binary.BigEndian.PutUint32(key, uint32(j))
+        }
+        f := NewWithEstimates(uint(b.N), 0.0001)
```

Comment:
Benchmarks should not change based on `b.N`, which probably doesn't even make sense in parallel benchmarks.